tests/data/fixture_data.py

```diff
@@ -117,6 +120,93 @@
         FIELD_ROOT_SAMPLE_ID: "MCM003",
         FIELD_RNA_ID: "rna_1"
     },
+    { # a 'limit of detection' result
+        FIELD_COORDINATE: "D01",
+        FIELD_SOURCE: "test1",
+        FIELD_RESULT: "limit of detection",
+        FIELD_PLATE_BARCODE: "123",
+        FIELD_COG_BARCODE: "klm",
+        FIELD_ROOT_SAMPLE_ID: "MCM004",
+        FIELD_RNA_ID: "rna_1"
+    },
+    { # positive, with low Ct values
+        FIELD_COORDINATE: "E01",
+        FIELD_SOURCE: "test1",
+        FIELD_RESULT: "Positive",
+        FIELD_PLATE_BARCODE: "123",
+        FIELD_COG_BARCODE: "nop",
+        FIELD_ROOT_SAMPLE_ID: "MCM005",
+        FIELD_RNA_ID: "rna_1",
+        FIELD_CQ_1: 5,
```

Comment:
Cq values are floats with about 8dp from the examples, so maybe make the tests use more realistic float numbers e.g. 5.32563219 etc.